### DISCUSSION TITLE
Module named config and Figspecs not found

### DISCUSSION THREAD

**Discussion by josht6022** [modflowpy/flopy#1270] [Q&A] [answered]
I am working on example 44. Zero-Order Production in a Dual-Domain System.
[(https://modflow6-examples.readthedocs.io/en/latest/_notebooks/ex-gwt-mt3dsupp632.htm)]

It seems that the **config** and **figspecs** libraries are not working.

Are they called something different now? Thank you for your response.
![image](https://user-images.githubusercontent.com/40776914/138177382-3ef6b58f-64ee-4b61-8fe0-43dd37c0c64f.png)

**Comment by christianlangevin** [comment, marked as answer]
This notebook runs fine as long as you download the entire modflow6-example repository and keep the notebooks in the notebooks directory.  Those two python files are located in a common folder, and this notebook sets a path to the common folder.  This is what I see when I run that notebook from its proper location, which suggests that everything is working as intended.

![image](https://user-images.githubusercontent.com/3372403/138280929-86838bb6-982a-46b5-9fee-ec9a29be86fa.png)

**Reply by ambernguyen13** [reply]
Hi Dr. Langevin, I am using Jupyter Lab with a screen just like this, but it is still not working for me :(. Do you have any suggestions as to what I am still doing wrong? Thank you!

**Reply by christianlangevin** [reply]
Hello @ambernguyen13, I'm sure it is something simple.  Do you see any error messages?  How do you know it is not working?

**Reply by ambernguyen13** [reply]
Hi Dr. Langevin,

This is what my screen looks like! 
<img width="960" alt="jupss" src="https://github.com/modflowpy/flopy/assets/90818175/737ef0e4-da33-463a-8da1-8ac1be5f1875">

Basically I still get the error "module named config and Figspecs not found."

Thank you!,
Amber N.

**Reply by ambernguyen13** [reply]
Oops sorry, I ran the wrong file, but I would like to get that issue resolved as well! This is what the screen looks like when I run the first MT3DMS example problem:
<img width="892" alt="jupss3" src="https://github.com/modflowpy/flopy/assets/90818175/96e1d526-ff75-4105-acec-eb33d77c4e26">

Thank you!,
Amber N.

**Comment by josht6022** [comment]
Hi,

Thank you for your quick response. I see what I did wrong. Also. I will be
using flopy for my phd research. Thank you for an amazing python code that
works with modflow.

Best,
Josh
